packages/api/tests/endpoints/collections/update-collection.js

```diff
@@ -119,35 +138,35 @@ test('PUT replaces an existing collection', async (t) => {
     .send(updatedCollection)
     .expect(200);
 
-  const fetchedDynamoRecord = await collectionModel.get({
+  const actualCollection = await collectionModel.get({
     name: originalCollection.name,
     version: originalCollection.version,
   });
 
-  t.is(fetchedDynamoRecord.name, originalCollection.name);
-  t.is(fetchedDynamoRecord.version, originalCollection.version);
-  t.is(fetchedDynamoRecord.duplicateHandling, 'error');
-  t.is(fetchedDynamoRecord.process, undefined);
+  const actualPgCollection = await t.context.collectionPgModel.get(knex, {
+    name: originalCollection.name,
+    version: originalCollection.version,
+  });
 
-  const fetchedDbRecord = await dbClient.first()
-    .from('collections')
-    .where({
-      name: originalCollection.name,
-      version: originalCollection.version,
-    });
+  t.like(actualCollection, {
+    ...originalCollection,
+    duplicateHandling: 'error',
+    process: undefined,
+    createdAt: originalCollection.createdAt,
+    updatedAt: actualCollection.updatedAt,
+  });
 
-  t.is(fetchedDbRecord.name, originalCollection.name);
-  t.is(fetchedDbRecord.version, originalCollection.version);
-  t.is(fetchedDbRecord.duplicate_handling, 'error');
-  // eslint-disable-next-line unicorn/no-null
-  t.is(fetchedDbRecord.process, null);
-  t.is(fetchedDbRecord.created_at.getTime(), fetchedDynamoRecord.createdAt);
-  t.is(fetchedDbRecord.updated_at.getTime(), fetchedDynamoRecord.updatedAt);
+  t.deepEqual(actualPgCollection, {
+    ...originalPgRecord,
+    duplicate_handling: 'error',
+    process: null,
+    created_at: originalPgRecord.created_at,
+    updated_at: actualPgCollection.updated_at,
```

Comment:
Just taking another look at this: does this test condition prove that the `updatedAt` value sent in the PUT request body **was not** used to set the value for the PG collection? We're pulling the `updatedAt` value from the PG collection in the database for comparison, but we're not really asserting anything about how that value was set.

Does this make sense? Do we need to also prove that `updated_at` does not equal the `updatedAt` value from the PUT request?

Reply:
It does - how about : bb12eb23f